src/modules/PublicLab.RichTextModule.js

```diff
@@ -196,6 +196,14 @@ module.exports = PublicLab.RichTextModule = PublicLab.Module.extend({
       }
     });
 
+                crossvent.add(_module.wysiwyg.textarea, 'keyup', function (e) {
+      _editor.validate();
+      var regexp= /[\*]{2}[\s]{0,1}[\n]+[\#]{3}[^\P{P}*]+[\*]{2}/;
```

Comment:
OK great, so I think we may need to add a comment inline describing what this segment of code does, since it's a bit mysterious looking. Just thinking ahead, thanks!